src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/jobsboard/api/EmployerRepositoryShould.kt

```diff
@@ -17,10 +17,10 @@ import org.springframework.test.context.ActiveProfiles
 import org.springframework.test.context.DynamicPropertyRegistry
 import org.springframework.test.context.DynamicPropertySource
 import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.jobsboard.api.commons.domain.EntityId
```

Comment:
These changes introduce a lot of noise on the PRs. I'd suggest next time, take note of it and do the refactor later, or do it in a different branch and rebase your current branch so the changes are more clear.

Reply:
okay noted, split the refactoring next time

Reply:
or shall I undo this and prepare another PR?
do-able for cleanness, again

Reply:
I undo and move these relocation (to commons) to future PR. Thanks

Reply:
updated

Reply:
For this PR we can keep these changes. It's OK :-)